src/engine/enginevumeter.cpp

```diff
@@ -93,6 +99,24 @@ void EngineVuMeter::process(CSAMPLE* pIn, const int iBufferSize) {
     } else {
         --m_peakDuration;
     }
+
+    if (clipped & 1) {
```

Comment:
My comment targets a slightly different issue but of cause a small. 
The clipped bitmap is the input. You calculate 3 cos from it while one is should be the result of the output of the L OR R :m_ctrlPeakIndicatorR || m_ctrlPeakIndicatorL.